### PR TITLE
Make single page app

### PR DESCRIPTION
**Pull request by SimonGrchevski**
- Add nav elements
- Add event listener to elements
- Change the content of main  on click on nav list item

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far! Well structured code, good UI, and nice readme.
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Use the Luxon library as required.
- [ ] Ensure you use the exact linter config rules for stylelint and eslintrc.json.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
+      screenArr[content].classList.add('display-page');
+    });
+  }
+  navItems.forEach((item, i) => {
+    item.addEventListener('click', () => display(i));
+  });
+  const bookListWrap = document.querySelector('.book-ul');
+  const addBtn = document.querySelector('.add');
+  const bookTitle = document.getElementById('title');
+  const bookAuthor = document.getElementById('author');
```
Comment (mikenath223) [inline js/main.js:60]: - [x] In the `function implementTime`, as required kindly use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library. See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time).

File: css/main.css [hunk 2]
```diff
@@ -240,3 +241,11 @@ footer {
...
   color: #fff;
 }
+
+.main-content {
+  display: none;
+}
+
+.display-page {
+  display: block;
+}
```
Comment (mikenath223) [inline css/main.css:251]: - [x] In both stylelintrc.json and eslitnrc.json you seem to be adding more rules than those stated in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript), kindly ensure you use the exact rules as they are shared by other students and keeps the project professional.

File: js/main.js (hunk 1 above)
Comment (Davidosky007) [reply js/main.js:60]: * I think while trying to fix the previous TSE requested changes you broke your code as the navigation links don't work please fix it 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @SimonGrchevski @vladanvi99, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
# _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mimipeshy — APPROVED** [review]
## Hi Team @SimonGrchevski and @vladanvi99 👋🏾 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**